docs/howto/proposals.md

```diff
@@ -0,0 +1,227 @@
+| `proposals` | A list of zero or more proposals from the Vega network. |
+
+<details><summary>Example response</summary>
+
+```js reference
+https://github.com/vegaprotocol/sample-api-scripts/blob/master/propose-votes/response-examples.txt#L2-L64
+```
+
+</details>
+
+:::info
+For full example code, please visit the [repo on GitHub](https://github.com/vegaprotocol/sample-api-scripts/blob/master/propose-votes/).
+:::
+
+## How do I vote for a proposal?
+
+Voting for a proposal requires the use of staking tokens in the wallet owned by the calling party (public key). The minimum amount required is set in the [network parameters](create-market.md#where-do-i-find-the-current-network-parameters) (see values ending in `minVoterBalance`).
+
+:::info
+At present the Vega testnet uses the **VEGA (testnet)** token for governance.  
+Additionally, when a Vega testnet network reset occurs, any user proposals including votes will be reset.
+:::
+
+For more information on how to check for token balances (including any used for governance/staking), please see the guide to retrieve party [positions and balances](positions-balances.md).
+
+### Log in to wallet and get public key
+
+See the section on the [Wallet service](wallet.md) to learn how to log in, list keys and select a public key.  
+
+:::info
+For a working **wallet example** used by this how-to guide, please visit the [API Samples GitHub](https://github.com/vegaprotocol/sample-api-scripts/blob/master/propose-markets/) repo.
+:::
+
+### Vote for Proposal
+
+After locating a proposal (`proposal_id`) using the listing calls shown above, to vote either **for** (YES) or **against** (NO), simply connect to a Vega API server and submit a vote for a *proposal*:
+
+<GitPod />
+
+<Tabs groupId="codesamples4">
+<TabItem value="python-rest" label="Python (REST)">
+
+```js reference
+https://github.com/vegaprotocol/sample-api-scripts/blob/master/propose-markets/propose-vote-enact-market.py#L151-L155
+```
+
+```js reference
+https://github.com/vegaprotocol/sample-api-scripts/blob/master/propose-markets/propose-vote-enact-market.py#L381-L389
+```
+
+```js reference
+https://github.com/vegaprotocol/sample-api-scripts/blob/master/propose-markets/propose-vote-enact-market.py#L396-L400
+```
+
+</TabItem>
+</Tabs>
+
+If successful, the response will include:
+
+| Field          |  Description  |
+| :----------------- | :------------- |
+| `signature` | A signed transaction message containing the vote data. In the same way as the market proposal message, if `propagate` is set to **true**, the signed data will be automatically forwarded by the wallet server to a node.
+
+<details><summary>Example response</summary>
+
+```js reference
+https://github.com/vegaprotocol/sample-api-scripts/blob/master/propose-markets/response-examples.txt#L19-L32
+```
+
+</details>
+
+
+
+## What's next?
+
+ * Learn about [Streaming events](event-stream.md)
+ * How to find and estimate [Trading fees](fees.md)
```

Comment:
In a follow up PR we can add a link to the proposal tutorials here. They provide shapes/examples for every type orf proposal.

Reply:
@candida-d vetoed this idea - the What's Next links are navigation between How-tos, so this would be confusing. THere might be somewhere else to add the link - but it's not here.